test/unit/specs/components/checkbox.spec.js

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/vue'
+import Checkbox from '~/components/Checkbox'
+import ByIcon from '~/assets/licenses/by.svg?inline'
+
+describe('Checkbox', () => {
+  it('should render a checkbox with a string label', () => {
+    render(Checkbox, { propsData: { id: 'simple', label: 'Is Code Poetry?' } })
+    // Get throws an error if element is not found, so this serves as an assert
+    screen.getByLabelText(/code/i, { checked: false })
```

Comment:
While these do throw errors if the element isn't found, I think for consistency and clarity it would still be nice to add an explicit assertion.

```suggestion
    const checkbox = screen.getByLabelText(/code/i)
    expect(checkbox).toHaveAttribute('checked', 'false')
```

For example